### PR TITLE
product page

### PR DESCRIPTION
**Pull request by prakashsakari**
1. Created Product Page
2. Integrated Mockebee API to get data

product-1

product-2

### CODE REVIEW

File: mockbee-test/ecom-store/src/pages/Products.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    const [products, setProducts] = useState([]);
+    const [error, setError] = useState();
+
+    useEffect(() => {
+        (async () => {
+            try{
+                const {data : {products}} = await axios.get("/api/products")
+                setProducts(products);
+            }catch(error){
+                setError("No products to display");
```
Comment (SanketDhabarde) [inline mockbee-test/ecom-store/src/pages/Products.js:14]: Where you are using this error state?

File: mockbee-test/ecom-store/src/components/ProductCard/ProductCard.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.product-content {
+    margin-top: 5rem;
+    margin-left: 20rem;
+    padding: 2rem 5rem;
+}
+  
+.card-image-container {
+    width: 100%;
+    height: 240px;
```
Comment (SanketDhabarde) [inline mockbee-test/ecom-store/src/components/ProductCard/ProductCard.css:9]: You can use `vh` units for height. or rems to keep it consistent.

File: mockbee-test/ecom-store/src/backend/db/products.js [hunk 3]
```diff
@@ -8,23 +8,105 @@ import { v4 as uuid } from "uuid";
...
-    price: "5000",
-    categoryName: "non-fiction",
+    imgUrl: "https://therightfit.netlify.app/assets/shoes.jpg",
+    isTrending: false,
+    isFavourite: false,
+    title: "Nike",
+    productCategory: "Men Sneakers",
+    newPrice: 1750,
+    oldPrice: 2499,
+    discount: Math.floor(Math.random() * 100)
```
Comment (SanketDhabarde) [inline mockbee-test/ecom-store/src/backend/db/products.js]: You can add `inStock` as well to product list

**Review by SanketDhabarde — APPROVED** [review]
Left a few suggestions. 
- You can also put images in a public folder.
- you can give `jsx` extension to components and `js` for any utils logic or constants. 
Everything looks good. I like your folder structure, like exporting form `index.js`

File: mockbee-test/ecom-store/src/pages/Products.js (hunk 1 above)
Comment (prakashsakari) [reply mockbee-test/ecom-store/src/pages/Products.js:14]: I have not used it yet, but have kept it for further use.
Thank you for the review.

File: mockbee-test/ecom-store/src/backend/db/products.js (hunk 3 above)
Comment (prakashsakari) [reply mockbee-test/ecom-store/src/backend/db/products.js]: I added inStock and Fast delivery both,

File: mockbee-test/ecom-store/src/components/ProductCard/ProductCard.css (hunk 2 above)
Comment (prakashsakari) [reply mockbee-test/ecom-store/src/components/ProductCard/ProductCard.css:9]: Sure will do this, generally is used vw, vh or rem but I think I missed replacing pixel here. Thank you for pointing out

File: mockbee-test/ecom-store/src/App.js [hunk 4]
```diff
@@ -1,11 +1,13 @@
 import "./App.css"
 import {Home} from "./pages/Home"
+import { Products } from "./pages/Products";
 import { Routes, Route } from "react-router-dom";
 export default function App() {
   return (
     <div className="App">
       <Routes>
         <Route path="/" element={<Home />} />
+        <Route path="products" element={<Products />} />
```
Comment (Aditi0505) [inline mockbee-test/ecom-store/src/App.js:10]: You can add / before products in path to ensure consistency.

**Review by Aditi0505 — APPROVED** [review]
Just left a small suggestion. 
1. The folder structure seems good
2. The components are well-defined
3.  Code is well readable.

File: mockbee-test/ecom-store/src/App.js (hunk 4 above)
Comment (prakashsakari) [reply mockbee-test/ecom-store/src/App.js:10]: Thank you for the review. I will make the changes.

**Pull request merged**